build.gradle.kts

```
@@ -56,17 +64,28 @@ tasks {
 }
 
 repositories {
-    jcenter()
+    mavenCentral()
 }
 
 dependencies {
-    api(group = "org.pf4j", name = "pf4j", version = "3.0.1")
-    testImplementation(platform("org.junit:junit-bom:5.7.0"))
+    val pf4jVersion: String by project
+    api(group = "org.pf4j", name = "pf4j", version = pf4jVersion)
+
+    val slf4jVersion: String by project
+    testImplementation(group = "org.slf4j", name = "slf4j-api", version = slf4jVersion)
+    
+    val junitVersion: String by project
+    api(platform("org.junit:junit-bom:${junitVersion}"))
```

Avant, c'était dans `testImplementation`, on est obligé de passer ça en `api` désormais ?
```suggestion
    testImplementation(platform("org.junit:junit-bom:${junitVersion}"))
```

---

C'est corrigé